.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: elixir
+elixir:
+  - 1.4
+addons:
+  postgresql: '9.4'
```

Comment:
9.6 was working fine for me on travis. (10 has issues though: https://github.com/travis-ci/travis-ci/issues/8537)